staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -1009,21 +1011,26 @@ func testCacherSendBookmarkEvents(t *testing.T, allowWatchBookmarks, expectedBoo
 
 func TestCacherSendBookmarkEvents(t *testing.T) {
 	testCases := []struct {
+		name                string
 		allowWatchBookmarks bool
 		expectedBookmarks   bool
 	}{
 		{
+			name:                "allowWatchBookmarks:true, expectedBookmarks: true",
 			allowWatchBookmarks: true,
 			expectedBookmarks:   true,
 		},
 		{
+			name:                "allowWatchBookmarks:false, expectedBookmarks: false",
 			allowWatchBookmarks: false,
 			expectedBookmarks:   false,
 		},
 	}
 
 	for _, tc := range testCases {
-		testCacherSendBookmarkEvents(t, tc.allowWatchBookmarks, tc.expectedBookmarks)
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
you don't parallelize those - did you want?

Reply:
I did in `testCacherSendBookmarkEvents` but i can move here so that it is more visible.